qml/MonitoringPanel.qml

```diff
@@ -265,52 +269,96 @@ ColumnLayout {
                     id: infoTabBar
+                    anchors.top: parent.top
+                    anchors.left: parent.left
+                    width: parent.width
                     TabButton {
                         text: qsTr("Info")
                     }
                     TabButton {
                         text: qsTr("Statistics")
                     }
-                    Layout.fillWidth: true
                 }
 
                 Rectangle {
                     id: infoSelectedEntity
-                    Layout.fillWidth: true
+                    property string app_id: "UNKNOWN_APP"
+                    anchors.top: infoTabBar.bottom
+                    anchors.left: parent.left
+                    width: parent.width
                     height: infoTabBar.height
-                    Label {
-                        id: infoSelectedEntityLabel
-                        text: "No entity selected"
-                        font.pointSize: 10
-                        font.italic: true
-                        anchors.verticalCenter: parent.verticalCenter
-                        anchors.horizontalCenter: parent.horizontalCenter
+
+                    Rectangle {
+                        id: app_logo_rect
+                        visible: infoSelectedEntity.app_id != "UNKNOWN_APP"
+                        anchors.top: parent.top
+                        anchors.left: parent.left
+                        width: parent.width
+                        height: infoSelectedEntity.app_id != "UNKNOWN_APP" ? (parent.width * logo_height_)/logo_width_ : 0
+                        color: "transparent"
+
+                        Image {
+                            id: app_logo
+                            smooth: true
+                            visible: parent.visible
+                            source: infoSelectedEntity.app_id != "UNKNOWN_APP"
+                                    ? "/resources/images/app_logos/" + infoSelectedEntity.app_id + ".png" : ""
+                            anchors.top: parent.top
+                            anchors.topMargin: logo_margin_
+                            sourceSize.width: parent.width
+                            sourceSize.height:parent.height
+                        }
+                    }
+
+                    Rectangle
+                    {
+                        color: "transparent"
+                        anchors.top: app_logo_rect.bottom
+                        height: infoTabBar.height
+                        width: parent.width
+
+                        Label {
+                            id: infoSelectedEntityLabel
+                            text: "No entity selected"
+                            font.pointSize: 10
+                            font.italic: true
+                            anchors.verticalCenter: parent.verticalCenter
+                            anchors.horizontalCenter: parent.horizontalCenter
+                            width: parent.width
```

Comment:
Apply this configuration to left panel - DDS Entities section.

Reply:
It would involve changing `MonitoringPanel`. and both `Physical` and `Logical` model view. I would rather change it in another pull request.